webapp/src/Controller/BoardBaseController.php

```diff
@@ -80,6 +81,10 @@ function ($player) use ($user) {
             return $player && $player->getIsCurrentPlayer() && $player->getIsUser();
         })) !== false;
 
+        $isHumanPlayer = $user && $state['gameState'] != 2 && current(array_filter($players, function ($player) {
```

Comment:
What is `2` meaning? use constant to explicit that (in general avoid "magic numbers" like that